proto/BUILD.bazel

```diff
@@ -1,13 +1,15 @@
-package(
-    default_visibility = ["//visibility:public"],
-    licenses = ["notice"],  # Apache 2.0
-)
-
+load("@bazel_skylib//rules:build_test.bzl", "build_test")
 load("@rules_proto//proto:defs.bzl", "proto_library")
 load("@com_github_grpc_grpc//bazel:cc_grpc_library.bzl", "cc_grpc_library")
 load("@com_github_grpc_grpc//bazel:python_rules.bzl", "py_grpc_library", "py_proto_library")
 load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 
+package(
+    default_visibility = ["//visibility:public"],
+    licenses = ["notice"],  # Apache 2.0
+    # copybara:uncomment default_applicable_licenses = ["//third_party/p4lang_p4runtime:license"],
```

Comment:
The issue is that the `LICENSE` file is in the root folder one level up the workspace. So we would need to copy the LICENSE file again to the proto/ folder.

Reply:
Right, because the WORKSPACE file sits in the `proto` folder instead of at `root` :/  https://github.com/p4lang/p4runtime/blob/main/proto/BUILD.bazel#L14-L15

@antoninbas do you think having two copies of the LICENSE file is a reasonable workaround?

Reply:
It sounds ok from my side, thanks for the changes. But let's hear from @antoninbas

Reply:
A symbolic link wouldn't work in this case? If not, or if it breaks something else, I have no issue with this change.

Reply:
Changed to symlink

Reply:
Would we be okay with making the symlink the other way around?
It will cause issues for us internally.
If that seems unreasonable, we will work around it internally somehow.

Reply:
The actual LICENSE file should probably be at the root of the repo.
But if this will create issues for you, it's fine to duplicate the file. No reason not to keep it simple.

Reply:
Yeah, makes sense. SGTM.